internal/handlers/common/typecode.go

```diff
@@ -138,3 +138,37 @@ func init() {
 		aliasToTypeCode[i.String()] = i
 	}
 }
+
+// AliasFromType returns type alias name for given value.
+func AliasFromType(v any) string {
+	switch v := v.(type) {
+	case *types.Document:
+		return "object"
+	case *types.Array:
+		return "array"
+	case float64:
+		return "double"
+	case string:
+		return "string"
+	case types.Binary:
+		return "binData"
```

Comment:
I wonder if we should use `typeCodeBinData.String()` instead to ensure that those strings are the same everywhere. The same in `hasSameTypeElements`. That is, assuming that those aliases _are_ the same

Reply:
Looks like MongoDB these aliases are the same. I have some ideas on how to simplify `hasSameTypeElements` but I suggest doing it in a separate PR.

Reply:
Ok, but can we simplify that new `AliasFromType` function? Because it looks like it started as a copy of `hasSameTypeElements` that was changed, and we went from two copies of "binData" string (one is generated by stringer, one is in `hasSameTypeElements`) to three. That breaks the Scout rule: "Always leave the code better than you found it". So no, we should replace "binData" with `typeCodeBinData.String()` in that PR

Reply:
Updated